oeps/oep-0046-arch-ddd-domains.rst

```diff
@@ -0,0 +1,327 @@
+
+Ubiquitous Language:
+  The set of terms and definitions that defines the entities and relationships within a Bounded Context. These terms may not have the same definitions across different Bounded Contexts.
+
+Anti-Corruption Layer:
+  A layer around the edge of a Bounded Context that converts entities and information retrieved from other Bounded Contexts in the same Subdomain into the Ubiquitous Language of this Bounded Context.
+
+Overall Behavior Flows
+======================
+
+A **learner** engages with a **Business Subdomain** to find learning content that meets their learning goals. They purchase that learning content via whatever mode is appropriate for that **Business Subdomain** (direct purchase, free enrollment, subscriptions, etc). The **Business Subdomain** requests fulfillment from the **Learner Subdomain** to allow the **learner** access to their selected learning content.
+
+An **educator** engages with the **Content Authoring** Subdomain to author learning content and make it available to the **Learning** subdomain.
+
+An **administrator** engages with the **Learning** Subdomain to set up learning content with specific dates, and with the **Business Subdomain** to group learning content created by **educators** in the **Content Authoring** Subdomain into units that match learners educational goals.
+
+
+The Big Picture
+===============
+
+.. image:: oep-0046/domains.svg
+
+
+Learning Content Subdomains
+===========================
+
+These subdomains are used to create and consume learning content
+
+Content Authoring
+-----------------
+**Core** Subdomain
+
+Content Authoring (core) allows educators to create, modify, discover, package, annotate (tag), and share learning  content.  Learning content may be packaged from multiple content sources.
+
+Users:
+  - Course teams
+  - Instructional designers
+
+
+Bounded Contexts
+~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 10 10 70
+
+   - - Bounded Context
+     - Type
+     - Description
+   - - Course Run Authoring
+     -	supporting
+     - Course Run Authoring (supporting) allows educators to launch a Course Run. This includes the ability to override configuration settings and fields, including dates and grading policy.
+   - - Learning Content Authoring
+     - core
+     - Learning Content Authoring (core) allows educators to create and publish learning content and pathways,  which can be utilized to create course runs.
+   - - Learning Content Repository
+     - supporting
+     - Learning Content Repository (supporting) provides a shared service for storing, versioning, sharing, and discovering published learning content.  The learning content is available at varying degrees of granularity - including Pathways, Blocks and Units.
```

Comment:
I'd recommend either dropping the word "shared", or helping explain why it is used here and not for other services.